### DISCUSSION TITLE
Replace 函数无效

### DISCUSSION THREAD

**Discussion by yihuan1221** [apache/doris#13060] [A - General / Q&A]
REPLACE (VARCHAR str, VARCHAR old, VARCHAR new)
str 是以要替换掉的字符串 old开头，replace函数就无法生效
select replace('old_str', 'old', 'new');
输出结果依旧是 old_str，期望值是输出结果为 new_str

**Comment by hf200012** [comment]
```
mysql> select replace('old_str', 'old', 'new');
+----------------------------------+
| replace('old_str', 'old', 'new') |
+----------------------------------+
| new_str                          |
+----------------------------------+
1 row in set (0.01 sec)
```

I test normal here, which version are you

**Reply by yihuan1221** [reply]
那应该就是版本的问题, 用的还是0.11
![image](https://user-images.githubusercontent.com/114655779/193178146-95a70a3f-67c3-4e3b-9d1b-59486689ab28.png)

**Reply by dataalive** [reply]
可以逐步升级上来了，咱们社区已经到1.1.2了，1.2也在路上了。
